package.json

```diff
@@ -15,6 +15,7 @@
   },
   "files": [
     "src",
+    "cli.js",
```

Comment:
Normally, `npm` should pick up `cli.js` from the `bin` entry. Did you notice a case where it didn't?